src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1631,6 +1631,8 @@ namespace winrt::TerminalApp::implementation
     //   or 0. (see TerminalPage::_ParseArgs)
     int32_t TerminalPage::SetStartupCommandline(winrt::array_view<const hstring> args)
     {
+        _appArgs.SetVersionString(ApplicationDisplayName(), ApplicationVersion());
```

Comment:
I'm a bit alarmed about this. Can AppArgs load up the static AppLogic singleton and cut out the middleman (and the setter in the idl, and the hstrings, etc.)?

Reply:
(like the settings serializer does)

Reply:
oh yea I forgot about that